lib/ace/http/service.ex

```diff
@@ -118,7 +118,7 @@ defmodule Ace.HTTP.Service do
             Keyword.take(@socket_options ++ options, [:mode, :packet, :active, :reuseaddr])
 
           {:ok, listen_socket} = :gen_tcp.listen(port, tcp_options)
-          Logger.info("Serving cleartext using HTTP/1")
+          Logger.info("Serving cleartext using HTTP/1 on port #{port}")
```

Comment:
You'll need to read the assigned port from the socket using `:gen_tcp` to handle the case when the port given is `0`.